runner.py

```diff
@@ -0,0 +1,21 @@
+import os
+import sys
+os.environ['DJANGO_SETTINGS_MODULE'] = 'xblock.test.settings'
+
+from django.conf import settings
+settings.configure(
+    DEBUG=True,
+    ALLOWED_HOSTS='*'
+)
+
+import django
+django.setup()
+
+# Now we instantiate a test runner...
+from django.test.utils import get_runner
+TestRunner = get_runner(settings)
+
+# And then we run tests and return the results.
+test_runner = TestRunner(verbosity=2, interactive=True)
+failures = test_runner.run_tests(['xblock.test.django.test_field_translation'])
+sys.exit(failures)
```

Comment:
It looks like this didn't quite work as expected in Travis.  I'll go ahead and start a pytest/pytest-django conversion PR, may be easier than trying to get this debugged.

Reply:
I agree. That's the direction we want to go anyway. As you saw, this technique requires making a separate test runner and running tests in two different ways. Using pytest alone along with pytest-django is better.